Add access-your-teaching-qualifications domain

NickGraham101 opened the pull request:
## Context

New service access-your-teaching-qualifications requires a DNS zone and FrontDoor profile

## Changes in this PR

- Adds DNS zone access-your-teaching-qualifications.education.gov.uk
- Adds FrontDoor profile
- Replaced script that creates resource group, storage account and key vault with the make recipe now in use on other TRA repos

## Guidance to review

The terraform will be ran locally using `make aytq domains-infra-apply` and `make aytq dev domains-apply`, please check config is correct.

NickGraham101 commented:
![](https://github.trello.services/images/mini-trello-icon.png) [Infrastructure and domain naming](https://trello.com/c/eT6yaaHD/694-infrastructure-and-domain-naming)

saliceti commented on Makefile:
```diff
@@ -11,50 +11,72 @@ dev:
+
+aytq:
+	$(eval DOMAINS_ID=aytq)
+	$(eval PRODUCT_NAME="Access Your Teaching Certificates")
```
Isn't it "Qualifications"?

saliceti commented on custom_domains/aytq/terraform.tf:
```diff
@@ -0,0 +1,25 @@
+  features {}
+  version         = "3.45.0"
+  alias           = "app_subcription"
+  subscription_id = var.app_sub_id
```
This may not work in the workflow since the service principal is tied to one subscription

saliceti replied:
As discussed IRL, this is deployed manually, not via pipeline

nickdevdfe commented:
All AYTQ environments successfully deployed. This PR is now ready to be reviewed and merged.

saliceti commented on Makefile:
```diff
@@ -6,75 +6,100 @@ help: ## Show this help
 
 .PHONY: dev
-dev:
+dev: ## set the dev enironment variables
```
Typo: enironment

nickdevdfe committed:
Added access-your-teaching-qualifications domain

Replaced PS script that creates resource group. Updated Makefile
for clearer instructions, and added a state argument to run a Terraform state.
